packages/test-runner-commands/src/sendKeysPlugin.ts

```diff
@@ -0,0 +1,41 @@
+import { TestRunnerPlugin } from '@web/test-runner-core';
+import type { ChromeLauncher } from '@web/test-runner-chrome';
+import type { PlaywrightLauncher } from '@web/test-runner-playwright';
+
+type SendKeysPayload = { type?: string; press?: string };
+
+export function sendKeysPlugin(): TestRunnerPlugin<SendKeysPayload> {
+  return {
+    name: 'send-keys-command',
+    async executeCommand({ command, payload = {}, session }): Promise<any> {
+      if (command === 'send-keys') {
+        // handle specific behavior for playwright
+        if (session.browser.type === 'playwright') {
+          const page = (session.browser as PlaywrightLauncher).getPage(session.id);
+          if (payload.type) {
+            await page.keyboard.type(payload.type);
+            return true;
+          } else if (payload.press) {
+            await page.keyboard.press(payload.press);
+            return true;
+          }
+        }
+
+        // handle specific behavior for puppeteer
+        if (session.browser.type === 'puppeteer') {
+          const page = (session.browser as ChromeLauncher).getPage(session.id);
+          if (payload.type) {
+            await page.keyboard.type(payload.type);
+            return true;
+          } else if (payload.press) {
+            await page.keyboard.press(payload.press);
+            return true;
+          }
+        }
+
+        // you might not be able to support all browser launchers
+        throw new Error(`Sending keys is not supported for browser type ${session.browser.type}.`);
```

Comment:
What happens for Web Driver tests? It'd be great to be able to do this on Sauce/BrowserStack too.

Reply:
I think we can support webdriver with this as well

Reply:
Not sure whether Web Driver has a similar API here or not, but I can check. Last I heard at least one of Sauce/BrowserStack was adding support for Playwright/Puppeteer, but I've not been keeping a close eye on that either. 

There are also a handful of places where there can/should be more features/options/API that I figured we'd do feature releases on if this worked enough to get learning on the scope and flexibility people would want form this sort of feature.

Reply:
For reference: https://webdriver.io/docs/api/browser/keys It's got a little more wiring up needed, but I'll dive a little deeper on this.

Reply:
We can make this a separate PR as well if it's a lot of effort. And like you said there are more places where we need to add more support.

Reply:
I'm not sure I can make time to follow the rabbit on this right now. Would love to get this in without if we all OK with that for now.